nn_sdwan/assets/service_checks.json

```
@@ -0,0 +1,15 @@
+[
+    {
+        "agent_version": "7.33.0",
+        "integration": "nn_sdwan",
+        "check": "nn_sdwan.sdwan_controller.online",
+        "statuses": [
+            "ok",
+            "warning",
+            "critical"
+        ],
+        "groups": [],
+        "name": "Netnology SDWAN Controller Availability",
```

Must match manifest display_name `Netnology SDWAN` 
```suggestion
        "name": "Netnology SDWAN",
```